hazelcast/src/test/java/com/hazelcast/cache/CacheBasicAbstractTest.java

```diff
@@ -534,7 +500,7 @@ public void testJSRCreateDestroyCreate()
         Cache<Integer, String> cache = cacheManager.createCache(cacheName, config);
         assertNotNull(cache);
 
-        Thread.sleep(1000);
+        sleepSeconds(1);
```

Comment:
I'm not sure why we have this sleep here. Is it safe to remove or are we waiting for some internal state of the created cache?